index.js

```diff
@@ -79,8 +80,54 @@ const handler = (req, res) => {
         });
 };
 
+const martha_v2_handler = (req, res) => {
+    var orig_url = req.body.url;
+    var pattern = req.body.pattern;
+    if(!orig_url) {
+        orig_url = JSON.parse(req.body.toString()).url;
+    }
+
+    var http_url = dosToHttps(orig_url)
+
+    console.log(http_url);
+    superagent.get(http_url)
+        .end(function(err, response) {
+            if(err) {
+                console.error(err);
+                res.status(502).send(err);
+                return;
+            };
+            try {
+                var parsedData = JSON.parse(response.text);
+            } catch(e) {
+                res.status(400).send(`Data returned not in correct format`);
+                return;
+            };
+            superagent
+                .get(`${config.bondBaseUrl}/link/v1/fence/serviceaccount/key`)
+                .set('authorization', req.get("authorization"))
+                .end(function(bondErr, bondResponse) {
+                    if(bondErr) {
+                        console.error(err);
+                        res.status(502).send(err);
+                        return;
+                    };
+                    try {
+                        var parsedServiceAccountKey = JSON.parse(bondResponse.text);
+                    } catch(e) {
+                        res.status(500).send(`Service account key not in correct format`);
+                        return;
+                    };
+                    res.status(200).send({'dos': parsedData, 'credentials': parsedServiceAccountKey})
```

Comment:
intent is to return json that looks like
{ "dos": <full response from resolution>, "credentials": <full response from bond>}
I am not certain this does that